### ISSUE TITLE
[Feature Request] Add Mudblazor designer

### ISSUE THREAD

**Issue by vgb1993** [MudBlazor/TryMudBlazor#59]
It would be great if we had a WYSIWYG designer. 

It doesen't need to be too advanced, just enought to quickly scaffold some code.
 
This would be a great help to some people who are getting started (like me) with Mudblazor.

Any thoughts?

**Comment by vgb1993** [comment]
Here's a project we could use as a starter

https://github.com/eyupalemdar/AlemdarLabs.FormDesigner

**Comment by henon** [comment]
You are right, it would be awesome to have something like that for MudBlazor. I guess it would be a lot of work though

**Comment by vgb1993** [comment]
Yes I can imagine 😅

That's why I linked this repo, I think it's a good start.
